inference-engine/src/mkldnn_plugin/utils/node_dumper.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#ifdef CPU_DEBUG_CAPS
+
+#include "node_dumper.h"
+
+#include "mkldnn_node.h"
+#include "utils/blob_dump.h"
+
+#include "ie_common.h"
+#include <array>
+#include <regex>
+#include <sstream>
+#include <string>
+
+using namespace InferenceEngine;
+
+namespace MKLDNNPlugin {
+
+void NodeDumper::setup() {
+    char* dumpDirEnv = getenv("OV_CPU_BLOB_DUMP_DIR");
+    if (dumpDirEnv)
+        dumpDirName = dumpDirEnv;
+
+    char* dumpAsTextEnv = getenv("OV_CPU_BLOB_DUMP_AS_TEXT");
+    if (dumpAsTextEnv)
+        shouldDumpAsText = true;
+
+    char* dumpInternalBlobsEnv = getenv("OV_CPU_BLOB_DUMP_INTERNAL_BLOBS");
```

Comment:
Not sure we need this, by ngraph migration we almost get rid of InternalBlobs structure. So I would propose to dump internal blobs by defaults.

Reply:
Done